### PR TITLE
Fixed creating bookmarks folder

### PR DESCRIPTION
**Pull request by godric3**
Quick fix for #14

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ document.addEventListener('DOMContentLoaded', ()=>{
   chrome.bookmarks.search({'title':'TabStache_base'}, (results)=> {
     if (results.length === 0) {
       chrome.bookmarks.getTree((tree)=>{
-          otherBookmarksID = tree[0].children[1].id;
+          let otherBookmarksID = tree[0].children[1].id;
```
Comment (dngrhm) [inline popup.js]: should this be const instead of let?

Comment (godric3) [inline popup.js]: I think you're right, will fix in a minute.

Comment (godric3) [inline popup.js]: done

**Comment by dngrhm** [comment]
Thanks!

**Pull request merged**